### PR TITLE
[WIP] Foldable devices with .NET MAUI

### PR DESCRIPTION
**Pull request by conceptdev**
Initial proposal for the **Foldable** NuGet docs.

- [x] What is the right location in the TOC? Currently in **Fundamentals** but could belong in **User Interface > Views** ?
- [ ] ~Whether to include `DualScreenInfo`?~
- [ ] Whether to include `Triggers`?
- [ ] More samples required?

### CODE REVIEW

File: docs/fundamentals/foldable/twopaneview.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "TwoPaneView foldable and large-screen layout control"
+description: "Learn how to use the SemanticProperties class in a .NET MAUI app, so that a screen reader can speak about the user interface elements on a page."
```
Comment (davidbritch) [inline docs/fundamentals/foldable/twopaneview.md]: Update required :)

File: docs/fundamentals/foldable/twopaneview.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+monikerRange: ">= net-maui-7.0"
+ms.date: 10/01/2022
+---
+# .NET MAUI TwoPaneView layout
+
+[![Browse sample.](~/media/code-sample.png) Browse the sample](https://github.com/conceptdev/dotnet-maui-samples/)
+
+The `TwoPaneView` class represents a container with two views that size and position content in the available space, either side-by-side or top-to-bottom. `TwoPaneView` inherits from `Grid` so the easiest way to think about these properties is as if they are being applied to a grid.
+
+![Surface Duo dual-screen emulator showing a basic TwoPaneView test app](media/emulator-foldable-maui-app.png)
```
Comment (davidbritch) [inline docs/fundamentals/foldable/twopaneview.md]: Afraid we need to use the `:::image` format now. See other docs for examples.

**Review by davidbritch — COMMENTED** [review]
Maybe add your alias/github to the metadata, so you get to see any issues opened by users.

**Comment by conceptdev** [comment]
Screwed up the branch, moved to #1057

**Pull request closed without merging**